core_backend/app/__init__.py

```diff
@@ -1,13 +1,28 @@
+from typing import Callable
+
 from fastapi import FastAPI
 from fastapi.middleware.cors import CORSMiddleware
+from prometheus_client import (
+    CollectorRegistry,
+    make_asgi_app,
+    multiprocess,
+)
 
 from .configs.app_config import DOMAIN, QDRANT_COLLECTION_NAME, QDRANT_VECTOR_SIZE
+from .prometheus_middleware import PrometheusMiddleware
 from .routers import admin, auth, manage_content, question_answer, whatsapp_qa
 from .utils import setup_logger
 
 logger = setup_logger()
 
 
+def make_metrics_app() -> Callable:
```

Comment:
See [this page](https://prometheus.github.io/client_python/exporting/http/fastapi-gunicorn/) for fastapi + gunicorn setup

Reply:
And this [page](https://prometheus.github.io/client_python/multiprocess/) for more details on multiproc setup

Reply:
Oh yeah. Will fix this